documentation/src/main/resources/pages/ditto/protocol-specification-topic.md

```diff
@@ -21,9 +21,9 @@ Examples for valid topic paths are:
 
 The entity's namespace in which the entity is located.
 
-## Entity ID
+## Entity Name
```

Comment:
When changing this heading, the link to it also changes.
Please check that no other page referenced to `protocol-specification-topic.html#entity-id` (adjust link otherwise).

Please also adjust in Line 10 the link + the text of the overall topic.

Reply:
You're right. I'll have a check :+1: